frontend/public/signup.css

```diff
@@ -0,0 +1,104 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background: url('https://rabblerouserteam.files.wordpress.com/2016/08/rabble-rouser-header.jpg') no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  font-family: 'Open Sans', sans-serif;
+}
+
+header {
+  background-color: #EA3B49;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding-left: 50px;
+}
+
+header img {
+  content:url('https://i.imgur.com/eecsJFb.png');
```

Comment:
I've just discovered that this doesn't work in firefox 😞 

Seems like the best cross-browser solution is to use an empty div with a background, which kinda sucks. Unless anyone else has other ideas?